esp-hal-common/src/analog/adc/riscv.rs

```diff
@@ -108,6 +108,16 @@ cfg_if::cfg_if! {
     }
 }
 
+// The number of analog IO pins, and in turn the number of attentuations,
+// depends on which chip is being used
+cfg_if::cfg_if! {
+    if #[cfg(esp32c6)] {
+        const NUM_ATTENS: usize = 7;
```

Comment:
Just an idea, what do you think about using constant in 'esp-hal-common/src/soc/esp32XX/mod.rs' instead of this config block?